app/components/post_attachment_opengraph/post_attachment_opengraph.js

```diff
@@ -248,15 +248,17 @@ export default class PostAttachmentOpenGraph extends PureComponent {
                         </Text>
                     </TouchableOpacity>
                 </View>
-                <View style={style.flex}>
-                    <Text
-                        style={style.siteDescription}
-                        numberOfLines={5}
-                        ellipsizeMode='tail'
-                    >
-                        {openGraphData.description}
-                    </Text>
-                </View>
+                {openGraphData.description &&
```

Comment:
In RSX this will still display if `openGraphData.description === ""`?

Reply:
No, it won't, but I don't think we want to show it since it'll just be a blank line